src/App.js

```diff
@@ -1,23 +1,25 @@
-import logo from './logo.svg';
-import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import './App.scss';
+import Navigation from './components/Navigation/Navigation.jsx';
+import Header from './components/Header';
+import ArticleList from './components/ArticlesList';
+import { useState } from 'react';
 
 function App() {
+
+  const [articles, setArticles] = useState([]);
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
+        <Route path="/" element={
+          <>
+            <Navigation />
+            <Header />
```

Comment:
If these are on every page, you can move when outside the `Routes`, but still in the app. They will then be in every page + whatever the routes renders